pkg/virt-controller/services/renderresources.go

```diff
@@ -257,6 +257,34 @@ func WithSEV() ResourceRendererOption {
 	}
 }
 
+func initMemoryRequest(vmi *v1.VirtualMachineInstance, overcommitment int) ResourceRendererOption {
+	return func(renderer *ResourceRenderer) {
+		if memRequest, exists := renderer.vmRequests[k8sv1.ResourceMemory]; exists && !memRequest.IsZero() {
+			return
+		}
+
+		resources := renderer.ResourceRequirements()
+
+		memory, err := util.GetVmiGuestMemory(vmi.Spec)
+		if err != nil {
+			// This should never happen
+			log.Log.Errorf("cannot initial memory request for vmi: %v", err)
+		}
+
+		if overcommitment == 100 {
+			resources.Requests[k8sv1.ResourceMemory] = memory
+		} else {
+			value := (memory.Value() * int64(100)) / int64(overcommitment)
```

Comment:
Silly question, but can the overcommitment ever be 0?

Reply:
It cannot be :slightly_smiling_face:  :
https://github.com/kubevirt/kubevirt/blob/b02b8fe998d85fcc7abd6271e8ecad4dd1f078fa/pkg/virt-config/configuration.go#L403